Azure/sonic-mgmt
# [pytest] Make test dir a Python package

### Description of PR
When having multiple conftest, dir leading to this conftest has
to be Python package (presence of `__init__.py`.) Also, adding
pytest basedir and current dir to PYTHONPATH.

signed-off-by: Tamer Ahmed <tamer.ahmed@microsoft.com>
Summary:
Fixes # (issue)

### Type of change


### Approach
#### How did you do it?
Added `__init__.py to various dirs.

#### How did you verify/test it?
```
pytest --testbed=vms6-t1-7060 --inventory=../ansible/str --testbed_file=../ansible/testbed.csv --host-pattern=str-a7060cx-acs-10 --module-path=../ansible/library --disable_loganalyzer --allow_recover --skip_sanity -vvvv --collect-only -m bsl
================================================================================================================================================== test session starts ===================================================================================================================================================
platform linux2 -- Python 2.7.12, pytest-4.6.9, py-1.8.1, pluggy-0.13.1 -- /usr/bin/python
cachedir: .pytest_cache
ansible: 2.8.7
rootdir: /var/host-acs-mgmt-repo/tests, inifile: pytest.ini
plugins: ansible-2.2.2
collected 672 items / 661 deselected / 11 selected                                                                                                                                                                                                                                                                       
<Module fdb/test_fdb.py>
  <Function test_fdb[ethernet]>
    1. verify fdb forwarding.
    2. verify show mac command on DUT for learned mac.
  <Function test_fdb[arp_request]>
    1. verify fdb forwarding.
    2. verify show mac command on DUT for learned mac.
  <Function test_fdb[arp_reply]>
    1. verify fdb forwarding.
    2. verify show mac command on DUT for learned mac.
<Module snmp/test_snmp_cpu.py>
  <Function test_snmp_cpu>
    Test SNMP CPU Utilization
    
    - Pulls CPU usage via shell commans
    - Polls SNMP for CPU usage
    - Difference should be < 2% (allowing float->int rounding on each result)
    
    
    TODO: abstract the snmp OID by SKU
<Module snmp/test_snmp_interfaces.py>
  <Function test_snmp_interfaces>
    compare the bgp facts between observed states and target state
<Module snmp/test_snmp_lldp.py>
  <Function test_snmp_lldp>
    Test checks for ieee802_1ab MIBs:
    - lldpLocalSystemData  1.0.8802.1.1.2.1.3
    - lldpLocPortTable     1.0.8802.1.1.2.1.3.7
    - lldpLocManAddrTable     1.0.8802.1.1.2.1.3.8
    
    - lldpRemTable  1.0.8802.1.1.2.1.4.1
    - lldpRemManAddrTable  1.0.8802.1.1.2.1.4.2
    
    For local data check if every OID has value
    For remote values check for availability for at least 80% of minigraph neighbors
    (similar to lldp test)
<Module snmp/test_snmp_psu.py>
  <Function test_snmp_numpsu>
  <Function test_snmp_psu_status>
<Module vlan/test_vlan.py>
  <Function test_vlan_tc1_send_untagged>
    Test case #1
    Verify packets egress without tag from ports whose PVID same with ingress port
    Verify packets egress with tag from ports who include VLAN ID but PVID different from ingress port.
  <Function test_vlan_tc2_send_tagged>
    Test case #2
    Send tagged packets from each port.
    Verify packets egress without tag from ports whose PVID same with ingress port
    Verify packets egress with tag from ports who include VLAN ID but PVID different from ingress port.
  <Function test_vlan_tc3_send_invalid_vid>
    Test case #3
    Send packets with invalid VLAN ID
    Verify no port can receive these pacekts

============================================================================================================================================= 661 deselected in 0.55 seconds =============================================================================================================================================
```
#### Any platform specific information?

#### Supported testbed topology if it's a new test case?

### Documentation
